plugins/statistics.koplugin/main.lua

```diff
@@ -280,31 +422,95 @@ function ReaderStatistics:getTotalStats()
            total_stats
 end
 
+
+function ReaderStatistics:getAdvStats()
+    local total_stats = {
+        {
+            T(_"Last Week"),"",
+            callback = function()
+                UIManager:show(KeyValuePage:new{
+                    title = T(_"Week Statistics"),
+                    kv_pairs = getDatesFromAll(7,"daily_weekday"),
```

Comment:
Nitpicking ;) It would be nice if you can keep the coding style consistent with the rest of the project, i.e. adding a space after the comma: `getDatesFromAll(7, "daily_weekday")`